src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+
+        User currentUser = userService.findByEmail(email);
```

Comment:
Let's call the findByEmail method only once. And try to solve this task in one line

Reply:
Done in the new commit